addons/api/addon/models/host-catalog.js

```diff
@@ -40,7 +41,8 @@ export const STATIC_CREDENTIAL_FIELDS = ['access_key_id, secret_access_key'];
 export default class HostCatalogModel extends GeneratedHostCatalogModel {
   // =attributes
 
-  #credentialType;
+  @tracked credentialTypeValue;
```

Comment:
Traditionally you would call this `_credentialType` before JS added private fields since it's not meant to be accessed directly

Reply:
oh thats a good idea! thank u!